doc/interfaces/tracking_flow.rst

```diff
@@ -0,0 +1,230 @@
+.. _tracking_flow:
+
+========
+Tracking
+========
+
+This tutorial walks through the steps to perform fiber tracking using DIPY.
+Multiple tracking methods are available in DIPY.
+
+You can try these methods using your own data; we will be using the data in
+DIPY. You can check how to :ref:`fetch the DIPY data<data_fetch>`.
+
+--------------------
+Local Fiber Tracking
+--------------------
+
+Local fiber tracking is an approach used to model white matter fibers by
+creating streamlines from local directional information. The idea is as follows:
+if the local directionality of a tract/pathway segment is known, one can
+integrate along those directions to build a complete representation of that
+structure. Local fiber tracking is widely used in the field of diffusion MRI
+because it is simple and robust.
+
+In order to perform local fiber tracking, three things are needed:
+
+#. A method for getting directions from a diffusion data set.
+#. A method for identifying when the tracking must stop.
+#. A set of seeds from which to begin tracking.
+
+We will be using the ``stanford_hardi`` dataset for the local tracking command
+line interface demonstration purposes.
+
+We will start by creating the directory to which to save the stopping file
+(e.g.: ``stop_file``)::
+
+    mkdir stop_file
+
+The workflow for each local tracking method requires the paths to the peaks and
+metrics files, stopping files, and seeding files. The default value for the
+local tracking method to be used (specified through the ``tracking_method``
+parameter) is eudx (EuDX tracking [Garyfallidis12]_). A number of optional
+arguments, such as the number of seeds per dimension within a voxel
+(``seed_density``), or the output directory (``out_dir``), can also be provided.
+
+To get the stopping file, we will create a binary file of the GFA of the CSA
+reconstruction method and set a stopping criterion by calling the ``dipy_mask``
+command::
+
+    dipy_mask recons-csa/gfa.nii.gz 0.25 --out_dir "stop_file"
+
+We will use the PVE WM map corresponding to the diffusion data being tracked,
```

Comment:
Reading this again, it may be worthwhile specifying what PVE and WM mean, e.g. "We will use the white matter (WM) partial volume effects (PVE) map corresponding to the (...)"

Reply:
Addressed in 74bf1c0